modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/views/table/TableCell.js

```diff
@@ -114,7 +114,7 @@ function TableCell({
 		(itemInlineChanges || inlineEditingSettings?.alwaysOn)
 	) {
 		return (
-			<ClayTable.Cell>
+			<DndTableCell columnName={String(options.fieldName)}>
```

Comment:
Yes, in this case fieldName can also contain an array so I cast it in a string to make it clear(er). 
It's odd and the property name should be changed but it would affect some code in other places so, for now, I preferred keeping it this way